src/common/vscode/message/MessageBuilder.ts

```diff
@@ -29,7 +30,8 @@ export default class MessageBuilder {
     /**
      * Return VS Code's built-in message function.
      */
-    private getShowMessageFunction(): (message: string, ...items: string[]) => Thenable<string | undefined> {
+    private getShowMessageFunction():
+        (message: string, options: vscode.MessageOptions, ...items: string[]) => Thenable<string | undefined> {
```

Comment:
Why did you create a private getter method for a function? Does your way any better than this one?

```suggestion
    private showMessage(message: string, options: vscode.MessageOptions, ...items: string[]) : Thenable<string | undefined> {
```

Reply:
You're right, I'll fix this